File: js/specialNumber.js
const specialNumber = n => {
  // turn number to string and split up individually
  let nString = n.toString().split("");
  let specialArray = [];

  // push results to special array
  for (let i = 0; i < nString.length; i++) {
    if (
      nString[i] === "0" ||
      nString[i] === "1" ||
      nString[i] === "2" ||
      nString[i] === "3" ||
      nString[i] === "4" ||
      nString[i] === "5"
    ) {
      specialArray.push(nString[i]);
    } else {
      return "NOT!!";
    }
  }
  // as long as length is not 0, it's special
  if (specialArray.length > 0) {
    return "Special!!";
  }
};

File: python/special_number.py
def special_number(number):
    # turn number into string
    number_str = str(number)
    # iterate through string
    for num in number_str:
        # if a character is greater than 5 break challenge
        if int(num) > 5:
            return "NOT!!"
    # makes it through the loop it's special
    return "Special!!"
